### PR TITLE
updated secp256k1 and hdwallet

### PR DESCRIPTION
**Pull request by fluidvanadium**

### CODE REVIEW

**Comment by zancas** [comment]
Why are we pointing at master?   Should we point at a specific immobile tag?

File: Cargo.toml [hunk 1]
```diff
@@ -44,7 +44,7 @@ futures = "0.3.15"
 futures-util = "0.3.28"
 getset = "0.1.2"
 group = "0.13"
-hdwallet = "0.4.1"
+hdwallet = { git = "https://github.com/fluidvanadium/hdwallet.git", branch = "master" }
```
Comment (zancas) [inline Cargo.toml]: Can we please make this a tag?

File: Cargo.toml [hunk 2]
```diff
@@ -44,7 +44,7 @@ futures = "0.3.15"
 futures-util = "0.3.28"
 getset = "0.1.2"
 group = "0.13"
-hdwallet = "0.4.1"
+hdwallet = { git = "https://github.com/fluidvanadium/hdwallet.git", tag = "update_secp256k1_to_0.27.0" }
```
Comment (zancas) [inline Cargo.toml:48]: Should this be a fork inside zingolabs?

Comment (fluidvanadium) [reply Cargo.toml:48]: doesnt matter. the important bit is that its a tag

**Comment by zancas** [comment]
Running tests locally..

**Comment by zancas** [comment]
CI fails on the `guppy` issue.  I will try resetting the cargo lock file..

**Comment by zancas** [comment]
I reset the Cargo.lock to `zingolabs/dev` Cargo.lock and ran:

`cargo check && cargo check --tests && cargo fmt && cargo clippy && ./utils/trailing-whitespace.sh reject`

I have pushed the resulting code (lock file), to see if CI passes in this context.

**Review by juanky201271 — APPROVED** [review]
nice.

**Pull request merged**